frontend/src/components/header/MenuOverlay.js

```diff
@@ -675,8 +681,18 @@ class MenuOverlay extends Component {
                     />
                   ))}
 
-                {queriedResults.length === 0 && query !== '' && (
-                  <span>There are no results</span>
+                {queriedResults.length === 0 &&
+                  query !== '' &&
```

Comment:
IMHO, the `query !== ''` expression is not needed.

Reply:
Actually it is needed otherwise you will see this message
<img width="721" alt="Screenshot 2022-01-24 at 09 45 11" src="https://user-images.githubusercontent.com/1708561/150741818-2816b127-0228-4a87-a833-045c3c1dd5ec.png">

Reply:
ahh, understand.... so `query !== ''` means "we already queried for some results".

Reply:
yep